gcovr/tests/test_gcovr.py

```diff
@@ -105,7 +109,7 @@ def compiled(request, name):
         assert run(['make', 'clean'], cwd=path)
 
 
-KNOWN_FORMATS = ['txt', 'xml', 'html', 'sonarqube', 'json', 'csv']
+KNOWN_FORMATS = ['txt', 'xml', 'html', 'css', 'sonarqube', 'json', 'csv']
```

Comment:
I'm not a fan of introducing a `css` output format for the tests since this makes the test suite slower, and the Makefiles become unintuitive. I understand that you're doing this in order to set an `OUTPUT_PATTERN`. But I think it would be better to expand the output patterns mechanism to allow for multiple patterns per type?

We need functionality to check CSS files in order to test the existing --no-html-self-contained option. But for this PR we can just output self-contained HTML files, without writing external CSS?

Reply:
Ok. I'll create a seperate PR to use a list of patterns.

Reply:
#383